jetson/odrive_bridge/src/__main__.py

```diff
@@ -289,7 +289,10 @@ def get_state_string(self):
 
 def publish_state_msg(msg, state_string):
     global odrive_controller_index
-    msg.state = state_string[:len(state_string) - len("_State")]
+    # Shortens the state string which is of the form "[insert_odrive_state]State"
+    # e.g. state_string is ErrorState, so short_state_string is Error
+    short_state_string = state_string[:len(state_string) - len("State")]

```

Comment:
oh wait sorry i thought ```:len(state_string) - len("State")``` that was an index ig was fine as before now w the added comment, up to you to change back or not